modules/dhcp/providers/server/isc.rb

```diff
@@ -409,4 +421,14 @@ def poap_options_statements(options)
       statements
     end
   end
+
+  # Get all IPv4 addresses provided by the ISC DHCP config line following the pattern "my-config-option-or-directive IPv4_ADDR[, IPv4_ADDR] [...];" and return an array.
+  def get_ip_list_from_config_line(option_line)
```

Comment:
It was already there, but i've added a test for `get_range_from_config_line` that i've extracted too.  I changed `loadSubnets` calls to `parse_config_for_subnets` as suggested.